README.md

```diff
@@ -47,12 +47,12 @@ This provider's versions are compatible with the following versions of Cluster A
 
 This provider's versions are able to install and manage the following versions of Kubernetes:
 
-|                                    | v1.16 | v1.17 | v1.18 | v1.19 | v1.20 | v1.21 |
-| ---------------------------------- | ----- | ----- | ----- | ----- | ----- | ----- |
-| OpenStack Provider v1alpha3 (v0.3) | ✓    | ✓    | ✓     | ✓    | ✓    |       |
-| OpenStack Provider v1alpha4 (v0.4) |       |       |       |       | ✓    | ✓     |
-| OpenStack Provider v1alpha4 (v0.5) |       |       |       |       | ✓    | ✓     |
-| OpenStack Provider v1alpha5        |       |       |       |       |       | ✓     |
+|                                    | v1.16 | v1.17 | v1.18 | v1.19 | v1.20 | v1.21 | v1.22 | v1.23 |
```

Comment:
Should we add CAPI v1.1 in l.40? Or change the v1.0 to v1.1?

Reply:
hmm, I changed it to `v1.1`, maybe it makes sense to remove the CAPI version here 🤔

Reply:
IMHO we must keep the CAPI version compatibility, so it's good as it is now 👍🏻